crates/sui-core/src/authority_server.rs

```diff
@@ -527,6 +527,7 @@ impl ValidatorService {
         Ok((tonic::Response::new(info), Weight::zero()))
     }
 
+    #[instrument(name= "ValidatorService::handle_submit_transaction", level = "debug", skip_all, err, fields(tx_digest = ?tracing::field::Empty))]
```

Comment:
Tracing with the #instrument macro ensure we also capture the returned error and also trace the whole stack